bin/ember-template-lint.js

```diff
@@ -157,7 +161,22 @@ function run() {
   for (let relativeFilePath of filesToLint) {
     let filePath = path.resolve(relativeFilePath);
     let fileName = relativeFilePath === STDIN ? filePathFromArgs : relativeFilePath;
-    let fileErrors = lintFile(linter, filePath, fileName.slice(0, -4));
+    let moduleId = fileName.slice(0, -4);
+    let fileErrors = lintFile(linter, filePath, moduleId);
+
+    if (printPending) {
+      let failingRules = fileErrors.reduce((memo, error) => {
+        if (memo.indexOf(error.rule) === -1) {
+          memo.push(error.rule);
+        }
+
+        return memo;
+      }, []);
```

Comment:
If you make this `new Set()` then you can avoid checking and manually deduping (and since `set.add` returns the `set` itself we can use a simpler one liner):

```js
      let failingRules = fileErrors.reduce((memo, error) => memo.add(error.rule), new Set());
```